includes/admin.form.inc

```diff
@@ -0,0 +1,36 @@
+<?php
+/**
+ * @file
+ * Admin callbacks for Islandora Videojs
+ */
+
+/**
+ * Admin form.
+ */
+function islandora_videojs_admin($form, &$form_state) {
+  // Get settings.
+  $form = array();
+  $form['hls_library'] = array(
```

Comment:
Usually these variables are prefixed by the module name so would be like `islandora_video_js_hls_library`.